web/src/components/ak-file-search-input.ts

```diff
@@ -0,0 +1,151 @@
+import "#elements/forms/SearchSelect/index";
+
+import { DEFAULT_CONFIG } from "#common/api/config";
+import { parseAPIResponseError, pluckErrorDetail } from "#common/errors/network";
+
+import { AKElement } from "#elements/Base";
+
+import { AKLabel } from "#components/ak-label";
+
+import { AdminApi, AdminFileListUsageEnum } from "@goauthentik/api";
+import { IDGenerator } from "@goauthentik/core/id";
+
+import { msg } from "@lit/localize";
+import { html } from "lit";
+import { customElement, property } from "lit/decorators.js";
+
+interface FileItem {
+    name: string;
+    url: string;
+    mime_type: string;
+    size: number;
+    usage: string;
+}
```

Comment:
This seems to duplicate a similar interface. Consider moving it along with the render functions into a `web/src/elements/utils/files.ts`